addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -160,4 +175,78 @@ options.registry.website_sale = options.Class.extend({
         }).then(this.reload);
     }
 });
+
+/**
+ * Handles the edition of snippet's products search bar.
+ */
+options.registry.productsSearchBar = options.Class.extend({
+    xmlDependencies: ['/website_sale/static/src/xml/website_sale.editor.xml'],
+
+    /**
+     * @override
+     */
+    start: function () {
+        this.searchProductsBar = this.$target.find('.o_wsale_products_search_bar_input');
+        this.searchProductsInput = this.$target.find('.o_wsale_search_products');
+        return this._super.apply(this, arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    onClone: function () {
+        this.searchProductsInput.attr({
+            'data-display-description': true,
+            'data-display-price': true,
+            'data-display-image': true,
+            'data-order-by': 'name asc',
+        });
```

Comment:
Why this ?

Reply:
Indeed, this seems useless. Is there any expected behavior on clone ? If not, I can remove that

Reply:
Besides cloning... no :) So I think this is not needed. The `onClone` method is for example required for elements with IDs, which cannot be duplicated (like you did with anchors in fact).